adapters/Unity/UnityAdapter/GADMAdapterUnitySingleton.m

```diff
@@ -154,26 +135,22 @@ - (void)presentInterstitialAdForViewController:(UIViewController *)viewControlle
   _currentShowingUnityDelegate = adapterDelegate;
 
   NSString *placementID = [adapterDelegate getPlacementID];
-  if ([UnityAds isReady:placementID]) {
-    UADSMediationMetaData *mediationMetaData = [[UADSMediationMetaData alloc] init];
-    [mediationMetaData setOrdinal:impressionOrdinal++];
-    [mediationMetaData commit];
-    [UnityAds show:viewController placementId:placementID];
-  } else {
-    UADSMediationMetaData *mediationMetaData = [[UADSMediationMetaData alloc] init];
-    [mediationMetaData setMissedImpressionOrdinal:missedImpressionOrdinal++];
-    [mediationMetaData commit];
-  }
+  [UnityAds show:viewController placementId:placementID];
 }
 
 #pragma mark - Unity Delegate Methods
 
 - (void)unityAdsPlacementStateChanged:(NSString *)placementId
                              oldState:(UnityAdsPlacementState)oldState
                              newState:(UnityAdsPlacementState)newState {
-  // This callback is not forwarded to the adapter by the GADMAdapterUnitySingleton and the adapter
-  // should use the unityAdsReady: and unityAdsDidError: callbacks to forward Unity Ads SDK state to
-  // Google Mobile Ads SDK.
+  id<GADMAdapterUnityDataProvider, UnityAdsExtendedDelegate> adapterDelegate;
+  @synchronized(_adapterDelegates) {
```

Comment:
No changes required now for this since the adapter uses synchronized elsewhere, but heads up that when we have more time at a later date we will consider refactoring this code to use dispatch per best practice #2 on https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/CONTRIBUTING.md#best-practices